.github/workflows/claude-pr-creation.yml

```
@@ -0,0 +1,31 @@
+  issue_comment:
+    types: [created]
+
+jobs:
+  create-pr:
+    if: contains(github.event.comment.body, '@claude')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: anthropics/claude-code-action@beta
+        with:
+          prompt: "${{ github.event.comment.body }}"
+          allowed_tools: [
```

actionlint (1.7.7)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)